assets/src/block-editor/Sidebar/AnalyticsTrackingSidebar.js

```diff
@@ -0,0 +1,59 @@
+import {PluginDocumentSettingPanel} from '@wordpress/edit-post';
+import {dispatch, useSelect} from '@wordpress/data';
+import {SelectSidebarField} from '../SidebarFields/SelectSidebarField';
+import {TextSidebarField} from '../SidebarFields/TextSidebarField';
+import {getSidebarFunctions} from './getSidebarFunctions';
+
+const {__} = wp.i18n;
+
+dispatch('core').addEntities([{
+  baseURL: '/planet4/v1/analytics-values',
```

Comment:
Don't we need to register this route in the theme too? Like we do in the plugin [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/classes/rest/class-rest-api.php#L392-L444)